internal/pkg/manifest/validate.go

```diff
@@ -2230,3 +2248,52 @@ func (i ImageLocationOrBuild) validate() error {
 	}
 	return nil
 }
+
+func validateConditionValuesPerRule(aliases []string, allowedSourceIPs []string) error {
+	if (len(aliases) >= maxConditionsPerRule) || (len(allowedSourceIPs) >= maxConditionsPerRule) ||
+		(len(aliases)+len(allowedSourceIPs) >= maxConditionsPerRule) {
```

Comment:
Could this be simplified to ⬇️ ?
```suggestion
	if len(aliases)+len(allowedSourceIPs) >= maxConditionsPerRule {
```